### PR TITLE
LiteX submodules update

### PR DESCRIPTION
**Pull request by kowalewskijan**
This PR bumps LiteX submodules and includes `generate.py` fixups to work with the latest LiteX.
PR can be merged after HW tests using generated bitstreams from litex examples are proven to work

### CODE REVIEW

File: xc/xc7/tests/soc/litex/generate.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class BaseSoC(SoCCore):
     def __init__(
             self,
             toolchain="vivado",
-            sys_clk_freq=int(60e6),
+            sys_clk_freq=int(100e6),
```
Comment (litghost) [inline xc/xc7/tests/soc/litex/generate.py:23]: It probably isn't a good idea to increase the sys clock freq.  I don't believe that VPR can achieve 100 MHz.  @acomodi ?

Comment (acomodi) [reply xc/xc7/tests/soc/litex/generate.py:23]: With latest LiteX we have a minimum of 80MHz, below which DDR does not work even in a full Vivado flow. I think that we can set it to 80MHz.

80MHz worked on HW with symbiflow, but might not work on every single arty.

**Review by litghost — COMMENTED** [review]
LGTM, @acomodi please review.  If green merge.

**Review by acomodi — APPROVED** [review]
LGTM. I'd wait for CI and check the bitstream produced on HW, to verify that the frequency increase does work.

**Comment by umarcor** [comment]
@kowalewskijan can you please rebase to solve the conflicts?

**Pull request merged**